libs/platform/src/lib/components/form/combo-box/combo-box.component.ts

```diff
@@ -228,7 +229,9 @@ export class ComboBoxComponent extends CollectionBaseInput {
     private patchQueryMethod() {
         if (this._comboBox) {
             this._comboBox.handleSearchTermChange = () => {
-                this.ds.match(this._comboBox.inputTextValue);
+                if (this.ds) {
```

Comment:
Why do you need this? DS cannnot be null, if it does they there is wrong logic and we dont initialize it at certain situation.

Reply:
> Why do you need this? DS cannnot be null, if it does they there is wrong logic and we dont initialize it at certain situation.

Just a good practice to check the null, and it was fialing when datasource was passed from the unit test script files

Reply:
new typescripts has something we that you can used xx!.variable and  xxx?.variable. As I said you dont need it here. If you they are usecases where DS is null let'f fix them. With this & for this specific case you are moving some potential issue somewhere else.